docker-compose.yml

```diff
@@ -128,7 +128,6 @@ services:
       retries: 5
     networks:
       - my-app
-  
```

Comment:
Please, revert changes from `docker-compose.yml` as they are unrelated to your task.